### ISSUE TITLE
[Learn Page Content] description of intro course is wrong

### ISSUE THREAD

**Issue by frankharkins** [Qiskit/qiskit.org#2133]
### Where is the problem?
On [qiskit.org/learn](https://qiskit.org/learn/?timeScale=1%20year&learnLevel=Beginner)

### What is the problem?
The card for the intro course says:
>This introduction course is around 3 hours long...

but:
- The content is closer to ~5 hours long
- The card also shows up under "1 year" for "time to spend learning".

### What would you expect?
- The text in the card to be changed to "5 hours" also need to change [here](https://qiskit.org/textbook-beta/course/introduction-course)
- The card to appear under "1 day / 1 week" (can't remember whether this is full or part time)

**Comment by amirebrahimi** [comment]
Additionally, if you play around with other filter settings like setting it to "1 week", the Introduction to Quantum Computing and Quantum Hardware course comes up, but that course is probably more appropriate for "1 month" since it originally happened over 2 weeks. 

Another example: Select "1 month" and then filter on "Beginner" and you'll see that the Introduction to Quantum Computing and Quantum Hardware shows up, which is definitely more advanced. And then if you select "Advanced" under "1 month", then it shows the Introduction Course. 

Basically, I think the logic in the filtering seems to have quite a few bugs.

**Comment by pandasa123** [comment]
@frankharkins, since we migrated the learn page, we should be able to close this issue?
